core-bundle/src/EventListener/SearchIndexListener.php

```diff
@@ -55,37 +56,55 @@ public function __construct(IndexerInterface $indexer, string $fragmentPath = '_
     public function __invoke(TerminateEvent $event): void
     {
         $request = $event->getRequest();
+        $response = $event->getResponse();
+
+        $document = Document::createFromRequestResponse($request, $response);
+
+        $needsIndex = $this->needsIndex($request, $response, $document);
+
+        try {
+            $success = $event->getResponse()->isSuccessful();
```

Comment:
Why is this inside the try block and `$needsIndex` is not?

Reply:
That's how it was before. My assumption is the try catch is only supposed to catch exceptions regarding `$event->getResponse()->isSuccessful();`?

Reply:
Can that throw anywhere? :D

Reply:
idk, you implemented it in https://github.com/contao/contao/commit/23630c179d6a46557edef3b278ab23fadc3168ed ;)

Reply:
kk, well just code style but Idc really :)